tests/e2e/volume_health_test.go

```diff
@@ -698,21 +722,20 @@ var _ = ginkgo.Describe("Volume health check", func() {
 
 	})
 
-	/*
-		Verify changing the annotated values on the PVC to random value
-
-		Steps
-		1. Create a Storage Class
-		2.	Create a PVC using above SC
-		3.	Wait for PVC to be in Bound phase
-		4.	Verify annotation added on the PVC in accessible
-		5.	Kubectl edit on the annotation of the PVC and change the annotation to inaccessible state
-		6.	Wait for the default time interval
-		7.	Verify health annotation is added on the PVC is accessible
-		8.	Delete PVC
-		9.	Verify PV entry is deleted from CNS
-		10.	Delete the SC
-	*/
+	// Verify changing the annotated values on the PVC to random value.
+	//
+	// Steps
+	// 1. Create a Storage Class.
+	// 2.	Create a PVC using above SC.
```

Comment:
You may want to align the lines.

Reply:
Good catch.

In my editor, a tab is 3 spaces, which is hard to detect this problem by going through the code.
Tabs are awkward. In C/C++, we simply avoid them.